File: foggle-client-sample/src/test/java/ClientTest.java
import org.foggle.client.FoggleClient;
import org.foggle.core.dto.FeatureDto;
import org.junit.jupiter.api.Test;

import java.io.IOException;
import java.util.List;

public class ClientTest {
	@Test
	public void getEnabledFeatures() throws IOException {
		FoggleClient client = new FoggleClient("http://localhost:42777");
		List<FeatureDto> features = client.getEnabledFeatures();
		System.out.println(features);
	}
}

File: foggle-kotlin-client/src/test/kotlin/ClientTest.kt
import org.foggle.client.FoggleClient
import org.junit.jupiter.api.Test

class ClientTest {
    @Test
    fun enabledFeatures() {
        val client = FoggleClient("http://localhost:42777")
        val features = client.allowedFeatures
        println(features)

        println(client.isEnabled("payments:modify-payment:amazing-feature"))
        println(client.isEnabled("something-disabled"))
    }
}
